files/2da/2da_bgee/statdesc.htm

```diff
@@ -8,204 +8,218 @@
-120         38133          ****
-121         23791          ****
-122         12029          ****
-123         8286           ****
-124         54332          ****
-125         22177          ****
-126         17398          ****
-127         12129          ****
-128         38594          ****
-129         54331          ****
-130         20438          ****
-131         54333          ****
-132         38577          ****
-133         2365           ****
-134         22903          ****
-135         12108          ****
-136         12125          ****
-137         54335          ****
-138         42870          ****
-139         34800          ****
-140         34768          ****
-141         54336          ****
-142         42865          ****
-143         42857          ****
-144         47835          ****
-145         12030          ****
-146         65598          ****
-147         63953          ****
-148         63941          ****
-149         63902          ****
-150         63904          ****
-151         70297          ****
-152         65434          ****
-153         63938          ****
-154         63922          ****
-155         64001          ****
-156         65438          ****
-157         63992          ****
-158         63993          ****
-159         63217          ****
-160         63842          ****
-161         63801          ****
-162         64324          ****
-163         64319          ****
-164         63873          ****
-165         63882          ****
-166        -1              ****
-168        -1              ****
-169        -1              ****
-170        -1              ****
-171        -1              ****
-172        -1              ****
-173        -1              ****
-174        -1              ****
-175        -1              ****
-176        -1              ****
-177        -1              ****
-178        -1              ****
-179        -1              ****
-180        -1              ****
-181        -1              ****
-182        -1              ****
-183        -1              ****
-184        -1              ****
-185        -1              ****
-186        -1              ****
-187        -1              ****
-188        25904           SPWI417D
-189        103062          SPPR150D
-190        103078          SPPR750D
-191        103117          SPSH004D
+<b><a name="0">0</a></b>           17391          ****
+<b><a name="1">1</a></b>           17392          ****
+<b><a name="2">2</a></b>           17393          ****
+<b><a name="3">3</a></b>           17394          ****
+<b><a name="4">4</a></b>           17395          ****
+<b><a name="5">5</a></b>           17396          ****
+<b><a name="6">6</a></b>           17397          ****
+<b><a name="7">7</a></b>           54337          ****
+<b><a name="8">8</a></b>           17399          ****
+<b><a name="9">9</a></b>           17400          ****
+<b><a name="10">10</a></b>           17401          ****
```

Comment:
You can see here you used the same amount of space between both columns, but the double digit numbers need one less for both to align. Same with three digit numbers later.

Reply:
Can that fix be automated? Or am I supposed to do that manually?

Reply:
Does your editor support regular expressions? Anyway, you're on mac, so should have sed and can try this in the terminal:
`sed -ir -e 's,(>..</a></b>) ,\1,' -e 's,(>...</a></b>)  ,\1,' files/2da/2da_bgee/statdesc.htm`

Reply:
> Does your editor support regular expressions? Anyway, you're on mac, so should have sed and can try this in the terminal:
> `sed -ir -e 's,(>..</a></b>) ,\1,' -e 's,(>...</a></b>) ,\1,' files/2da/2da_bgee/statdesc.htm`

Yes, my editor support regex (VSCode).
Got this error:
```
sed: 1: "s,(>..</a></b>) ,\1,
": \1 not defined in the RE
```

I'm also trying messing with `style` by setting `text-align` and/or `white-space`, but still nothing...
Something like `<pre style="text-align: justify; white-space: pre-wrap;">`...

Reply:
That's suspicious, but try: `sed -i -e 's,\(>..</a></b>\) ,\1,' -e 's,\(>...</a></b>\) ,\1,' files/2da/2da_bgee/statdesc.htm`

or with the individual expressions in your editor's search&replace.

The whole point of pre is to keep things PREformatted, so styling should be limited, relayouting excluded.

Reply:
> That's suspicious, but try: `sed -i -e 's,\(>..</a></b>\) ,\1,' -e 's,\(>...</a></b>\) ,\1,' files/2da/2da_bgee/statdesc.htm`
> 
> or with the individual expressions in your editor's search&replace.
> 
> The whole point of pre is to keep things PREformatted, so styling should be limited, relayouting excluded.

Now I got
`-e: No such file or directory`

Anyway, I'll go regex replacement route in VSCode...

Reply:
I guess you have a bsd sed or something. The individual search expressions are between the 2nd and 3rd commas, while the replacement is just \1.

Reply:
@lynxlynxlynx 

So, I ended up making an invisible `<table>`, i.e.: `<td style="border: 0; font-family: monospace;">`.